msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/msa/AbstractMsaBrokerTest.kt

```diff
@@ -37,10 +37,6 @@ import org.junit.Before
  * Parent class for Brokered MSA tests. Ensures PRTv3 is enabled before tests are run
  */
 abstract class AbstractMsaBrokerTest : AbstractMsalBrokerTest() {
-    @Before
```

Comment:
can I have more information about why this is removed?

Reply:
prtv3 is enabled by default now, right?

Reply:
Yes, PRTV3 flight is removed now. It is enabled by default.